setup.py

```diff
@@ -7,11 +8,16 @@
 PACKAGENAME = "galsampler"
 VERSION = "0.0.dev"
 
+_THIS_DRNAME = os.path.dirname(os.path.abspath(__file__))
+REQUIREMENTS_FN = os.path.join(_THIS_DRNAME, "requirements.txt")
+with open(REQUIREMENTS_FN, "r") as f:
+    DEPENDENCIES = [req.strip() for req in f]
```

Comment:
This is not generally a good idea. For example, you don't need cython at runtime but you will need it when building.